src/compute/compute.py

```diff
@@ -814,7 +822,10 @@ def cancel_job_task(auth_header,system_name, system_addr,action,task_id):
 
     # if "error" word appears:
     if in_str(data,"error"):
-        update_task(task_id, auth_header, async_task.ERROR, data)
+        # error message: "scancel: error: Kill job error on job id 5: Invalid job id specified"
+        # desired output: "Kill job error on job id 5: Invalid job id specified"
+        err_msg = data[(data.index("error")+7):]
```

Comment:
error is such a common word in error messages that this risks unintentionally prunning error mesages.
I'd rather look for the specific prefix you want to cleanup (something like `scancel: error:` or something more specific like that)

Since this is not storage specific, and I think I've seen something like this before (IIRC in `cscs_api_common`), I'd consider moving this check to a more visible/reusable place.

Reply:
This is specific from `DELETE compute/jobs` so there shouldn't be another occurrence. We could be more specific on the search of the string, but since `str.index(s)` captures the first occurrence of a substring it should be enough.